contracts/KinTokenSale.sol

```diff
@@ -78,6 +78,7 @@ contract KinTokenSale is Ownable, TokenHolder {
     address[] public tokenGrantees;
     mapping (address => TokenGrant) public tokenGrants;
     uint256 public lastGrantedIndex = 0;
+    uint256 public constant MAX_TOKEN_GRANTEES = 100;
```

Comment:
this name is a bit misleading. it should imply that this is only used when adding/removing grants. Maybe rename to `MAY_TOKEN_GRANTEE_BATCH_SIZE`?